src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,42 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static #animalsList = [];
+
+  constructor(name, health = 100) {
+    this.health = health;
+    this.name = name;
+    Animal.#animalsList.push(this);
+  }
+
+  static get alive() {
+    return Animal.#animalsList.filter((animal) => animal.health > 0);
+  }
 }
 
 class Herbivore extends Animal {
-  // write your code here
+  constructor(name, health = 100) {
+    super(name, health);
+    this.hidden = false;
+  }
+
+  hide() {
```

Comment:
The method `hide` is implemented correctly according to the task requirements. It sets the `hidden` property to `true` without toggling it.